dask/bytes/tests/test_s3.py

```diff
@@ -283,9 +283,17 @@ def test_read_text_passes_through_options():
         assert df.count().compute(get=get) == 3
 
 
-def test_parquet(s3):
+@pytest.mark.parametrize("engine", [
+    'auto',
+    'pyarrow',
```

Comment:
In the current version the dask engine keyword is `'arrow'`, not `'pyarrow'`. This is fixed in #2868, but for now you'll need to change the engine name.